docs/index.html

```diff
@@ -37,11 +37,6 @@ <h2 class="bd-featurette-title">Easy to get started.</h2>
 {% endhighlight %}
       </div>
     </div>
-
-    <hr class="half-rule m-t-0">
-
-    <p><strong>Looking for something simpler?</strong> Customized builds of Bootstrap's CSS and JS are also available.</p>
-    <a href="{{ site.baseurl }}/getting-started/download/#custom-builds" class="btn btn-bs btn-outline">More download options</a>
```

Comment:
The fragment is broken and the text sounded like it was related to the removed customiser, so I just removed it. I'll take this change out and let you decide what you want to do with it.